client/apps/torii-launcher/src/frontend/SyncingState.tsx

```
@@ -0,0 +1,99 @@
+        }
+        setCurrentToriiBlock(currentBlock);
+      } catch (error) {
+        console.error("Error getting torii current block", error);
+      }
+    }, SYNC_INTERVAL);
+    return () => {
+      clearInterval(interval);
+    };
+  }, [reset]);
+
+  return isNaN(progress) || gameSynced ? (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)